### ISSUE TITLE
Maintainers wanted

### ISSUE THREAD

**Issue by cleishm** [tomlj/tomlj#44]
I wrote TOMLJ at a time when I was using it fairly actively. I'd love to see it maintained, but that needs maintainers who are willing to submit pull requests. Seems a lot of people like the library, so please feel free to get involved 😀

**Comment by cleishm** [comment]
@lkishalmi The main branch was basically complete in implementing TOML 1.0.0, sans a few edge cases. So I have been spending a little time getting a new release out. However, I am not likely to maintain the project over the long term and new maintainers are encouraged to join.

@kPYKx7Ddw4n1aIKZ has been contributing recently, which has been greatly appreciated!

**Comment by ebresie** [comment]
Is there any new things expected and needed work on?

**Comment by A248** [comment]
Yes, @ebresie there seem to be remaining open questions and issues with serialization. For example, #57 suggests a bug in the implementation. Also, #58 and #59 suggest the need for a better serialization API, especially a type hierarchy for toml values.
